src/stactools/cli/commands/copy.py

```diff
@@ -78,12 +78,21 @@ def create_copy_command(cli: click.Group) -> click.Command:
             "instead of the destination folder."
         ),
     )
+    @click.option(
+        "--skip-resolve",
```

Comment:
Maybe it's because I came up with the `skip_unresolved` flag in **pystac**, but this name was (slightly) surprising. What do you think about `--do-not-resolve`?

I don't feel that strongly on this, so if you prefer `--skip-resolve` I'm ok w/ keeping it.

Reply:
yeah I kind of was riffing on `skip_unresolved` and `resolve_links`. `resolve_links` is really the word I want but I wanted to negate it. How about `no-resolve`.

Reply:
Ok I learned about a pattern in click that is pretty good. See what you think:

```sh
  --resolve-links / --no-resolve-links
                                  Whether to resolve HREF links. Defaults to
                                  --resolve-links. Use --no-resolve-links to
                                  avoid writing external child objects
                                  locally.

```